db/seeds.rb

```diff
@@ -5,6 +5,10 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Manufacturer.destroy_all
+Vehicle.destroy_all
+Dealership.destroy_all
```

Comment:
Thanks for following up on this conversation. I'm glad we have this in seeds.rb now.